### PR TITLE
Jinghan/test streaming features in online store

### PR DESCRIPTION
**Pull request by jinghancc**
This PR does:
- unify streaming feature samples with batch feature samples in unit tests
- add test cases for streaming features in online method `Get`

### CODE REVIEW

File: internal/database/online/test_impl/get.go [hunk 1]
```diff
@@ -1,78 +1,86 @@
...
+				Features:  s.Features,
+			}
+			if s.Group.Category == types.CategoryBatch {
+				opt.RevisionID = &s.Revision.ID
+			}
+			rs, err := store.Get(ctx, opt)
+			require.NoError(t, err)
+
+			for i, f := range s.Features {
+				fmt.Println(rs[f.FullName()])
```
Comment (YikSanChan) [inline internal/database/online/test_impl/get.go:34]: rm?

**Review by YikSanChan — APPROVED** [review]
lgtm

**Review by lianxmfor — APPROVED** [review]
lgtm

File: internal/database/online/test_impl/get.go (hunk 1 above)
Comment (jinghancc) [reply internal/database/online/test_impl/get.go]: fixed

**Pull request merged**